build/scripts/build-client-binary

```diff
@@ -128,6 +128,11 @@ main() {
 
     rm -rf ${binary_tarball}
     mkdir -p $(dirname ${binary_tarball}) || :
+    
+    if [[ $keep_pmm_binary != "1" ]]; then
```

Comment:
If `pmm_version` is global, then we could put the code here from https://github.com/percona/pmm/pull/1207/files#diff-f6033d471a2409e0d14b46e8bb0d7ad855b5093ff9c109246c388706e819ed76R53-R55